index.html

```diff
@@ -1265,6 +1268,15 @@ <h3>Video format support (last update July 2020)</h3>
             }
           },
 
+	  copySubtitle: function () {
+	    var helpertextarea = document.createElement("textarea");
+	    document.body.appendChild(helpertextarea);
+	    helpertextarea.value = document.getElementsByClassName('current-caption')[0].textContent;
+	    helpertextarea.select();
```

Comment:
Not sure if this is intentional, but this'll deselect anything you already have selected, which, I actually don't mind that much now that I think about it out loud. Might be useful for quickly deselecting since there actually aren't that many places you can click on the site and have your text selection go away. I'm okay with this.

Reply:
It's just the simplest way I've found to copy text to the clipboard, but if you don't mind it then that's good.